modules/apps/commerce/headless/headless-commerce/headless-commerce-delivery-cart-impl/src/main/java/com/liferay/headless/commerce/delivery/cart/internal/resource/v1_0/CartResourceImpl.java

```diff
@@ -755,6 +755,9 @@ private void _updateOrder(CommerceOrder commerceOrder, Cart cart)
 			commerceOrder.getTotalDiscountAmount(),
 			commerceOrder.getTotalWithTaxAmount(), commerceContext, true);
 
+		_commerceOrderService.updatePrintedNote(
+			commerceOrder.getCommerceOrderId(), cart.getPrintedNote());
+
```

Comment:
Still not correct.
```
			commerceOrder = _commerceOrderService.updatePrintedNote(
				commerceOrder.getCommerceOrderId(),
				GetterUtil.getString(
					order.getPrintedNote(), commerceOrder.getPrintedNote()));
```

Once you made this change, you can send it to liferay-commerce and assign it to me there.

Reply:
@brianikim In the GetterUtil, it should be `cart.getPrintedNote()` instead of `order.getPrintedNote()` right? Also, why assign it to a variable if it isn't gonna be used? just Liferay convention?

Reply:
Sorry, yes cart.getPrintedNote() is correct. Also, what do you mean it isn't used? The updated commerceOrder will be used in line 768 and 773.

Reply:
oh i hadn't seen that. Alright, i'll make the changes. Thank you!